device/mellanox/x86_64-mlnx_msn2700-r0/plugins/eeprom.py

```diff
@@ -34,6 +35,20 @@ def log_error(msg):
     syslog.syslog(syslog.LOG_ERR, msg)
     syslog.closelog()
 
+
+def get_platform():
+    with open('/host/machine.conf', 'r') as f:
+        lines = f.readlines()
+        platform_line = filter(lambda x: 'onie_platform' in x, lines)[0]
+        return platform_line.strip('onie_platform=').strip('\n')
+
+
+platform = get_platform()
+if 'simx' in platform:
+    platform_path = os.path.join('/usr/share/sonic/device', platform)
+    subprocess.Popen(['/usr/bin/xxd', '-r', '-p', 'syseeprom.hex', 'syseeprom.bin'], cwd=platform_path)
```

Comment:
@mykolaf it's better to check the return code here. Can we use check_call? Also i think we can add some condition to avoid fake EEPROM regeneration in case it was already done.